testament/important_packages.nim

```diff
@@ -1,159 +1,156 @@
-pkg2 "nimgen", "nim c -o:nimgenn -r src/nimgen/runcfg.nim"
-pkg2 "nimlsp"
-pkg2 "nimly", "nim c -r tests/test_readme_example.nim"
-# pkg2 "nimongo", "nimble test_ci"
-# pkg2 "nimph", "nimble test", "https://github.com/disruptek/nimph"
-pkg2 "nimpy", "nim c -r tests/nimfrompy.nim"
-pkg2 "nimquery"
-pkg2 "nimsl"
-pkg2 "nimsvg"
-pkg2 "nimterop", "nimble minitest"
-pkg2 "nimwc", "nim c nimwc.nim"
-# pkg2 "nimx", "nim c --threads:on test/main.nim"
-# pkg2 "nitter", "nim c src/nitter.nim", "https://github.com/zedeus/nitter"
-pkg2 "norm", "nim c -r tests/sqlite/trows.nim"
-pkg2 "npeg", "nimble testarc"
-pkg2 "numericalnim", "nim c -r tests/test_integrate.nim"
-pkg2 "optionsutils"
-pkg2 "ormin", "nim c -o:orminn ormin.nim"
-pkg2 "parsetoml"
-pkg2 "patty"
-pkg2 "pixie", useHead = false
-pkg2 "plotly", "nim c examples/all.nim"
-pkg2 "pnm"
-pkg2 "polypbren"
-pkg2 "prologue", "nimble tcompile"
-pkg2 "protobuf", "nim c -o:protobuff -r src/protobuf.nim"
-pkg2 "pylib"
-pkg2 "rbtree"
-pkg2 "react", "nimble example"
-pkg2 "regex", "nim c src/regex"
-pkg2 "result", "nim c -r result.nim"
-pkg2 "RollingHash", "nim c -r tests/test_cyclichash.nim"
-pkg2 "rosencrantz", "nim c -o:rsncntz -r rosencrantz.nim"
-pkg2 "sdl1", "nim c -r src/sdl.nim"
-pkg2 "sdl2_nim", "nim c -r sdl2/sdl.nim"
-pkg2 "sigv4", "nim c --gc:arc -r sigv4.nim", "https://github.com/disruptek/sigv4"
-pkg2 "snip", "nimble test", "https://github.com/genotrance/snip"
-pkg2 "strslice"
-pkg2 "strunicode", "nim c -r src/strunicode.nim"
-pkg2 "synthesis"
-pkg2 "telebot", "nim c -o:tbot -r src/telebot.nim"
-pkg2 "tempdir"
-pkg2 "templates"
-pkg2 "tensordsl", "nim c -r tests/tests.nim", "https://krux02@bitbucket.org/krux02/tensordslnim.git"
-pkg2 "terminaltables", "nim c src/terminaltables.nim"
-pkg2 "termstyle", "nim c -r termstyle.nim"
-pkg2 "timeit"
-pkg2 "timezones"
-pkg2 "tiny_sqlite"
-pkg2 "unicodedb", "nim c -d:release -r tests/tests.nim"
-pkg2 "unicodeplus", "nim c -d:release -r tests/tests.nim"
-pkg2 "unpack"
-pkg2 "websocket", "nim c websocket.nim"
-# pkg2 "winim"
-pkg2 "with"
-pkg2 "ws"
-pkg2 "yaml", "nim build"
-pkg2 "zero_functional", "nim c -r -d:nimWorkaround14447 test.nim"
-pkg2 "zippy"
+pkg "hts", "nim c -o:htss src/hts.nim"
+# pkg "httpauth"
+pkg "illwill", "nimble examples"
+pkg "inim"
+pkg "itertools", "nim doc src/itertools.nim"
+pkg "iterutils"
+pkg "jstin"
+pkg "karax", "nim c -r tests/tester.nim"
+pkg "kdtree", "nimble test", "https://github.com/jblindsay/kdtree"
+pkg "loopfusion"
+pkg "macroutils"
+pkg "manu"
+pkg "markdown"
+pkg "memo"
+pkg "msgpack4nim", "nim c -r tests/test_spec.nim"
+pkg "nake", "nim c nakefile.nim"
+pkg "neo", "nim c -d:blas=openblas tests/all.nim"
+# pkg "nesm", "nimble tests" # notice plural 'tests'
+# pkg "nico"
+pkg "nicy", "nim c -r src/nicy.nim"
+pkg "nigui", "nim c -o:niguii -r src/nigui.nim"
+pkg "nimcrypto", "nim r --path:. tests/testall.nim" # `--path:.` workaround needed, see D20210308T165435
```

Comment:
> see D20210308T165435

What is "D20210308T165435" and where can I see it?

Reply:
the rationale for those "obfuscated" names is explained in https://github.com/timotheecour/Nim/issues/596